go/worker/merge/committee/node.go

```diff
@@ -372,9 +371,10 @@ func (n *Node) tryFinalizeResultsLocked(pool *commitment.Pool, didTimeout bool)
 	// to consensus and not change the internal Discrepancy flag.
 	cid := pool.GetCommitteeID()
 	consensusTimeout := state.consensusTimeout[cid]
+	runtimeTimeout := n.commonNode.Runtime.MergeRoundTimeout
```

Comment:
This should be the compute round timeout as `pool` is the per-compute-committee pool.